src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/HelperTests/LanguageHelperTest.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Altinn.Platform.Storage.Helpers;
+using Xunit;
+
+namespace Altinn.Platform.Storage.UnitTest
+{
+    public class LanguageHelperTest
+    {
+
+        [Fact]
```

Comment:
Consider using Theory instead. This would allow for input to the test method with the help of InlineDataAttribute.

Reply:
Neat, that made it a lot cleaner. Thanks!